lumigator/backend/backend/services/workflows.py

```diff
@@ -152,14 +153,18 @@ async def _run_inference_eval_pipeline(
 
             outputs = RunOutputs(
                 metrics={
-                    "rouge1_mean": round(eval_output.rouge.rouge1_mean, 3),
-                    "rouge2_mean": round(eval_output.rouge.rouge2_mean, 3),
-                    "rougeL_mean": round(eval_output.rouge.rougeL_mean, 3),
-                    "rougeLsum_mean": round(eval_output.rouge.rougeLsum_mean, 3),
-                    "bertscore_f1_mean": round(eval_output.bertscore.f1_mean, 3),
-                    "bertscore_precision_mean": round(eval_output.bertscore.precision_mean, 3),
-                    "bertscore_recall_mean": round(eval_output.bertscore.recall_mean, 3),
-                    "meteor_mean": round(eval_output.meteor.meteor_mean, 3),
+                    "rouge1_mean": round(eval_output.metrics["rouge"]["rouge1_mean"], 3),
```

Comment:
Let's think about making this more generic in a future PR. I don't know exactly how, and it'll depend on how the rest of the system develops, but let's keep an eye on this.

Reply:
Absolutely! I think we've got some more work to do on how we specify and store metrics.